WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -40,7 +40,7 @@ export default function Delegates() {
 
   const activeRegion = React.useMemo(() => {
     const selectedRegionIndex = delegateRegions.findIndex((region) => region.id === +hash);
-    if (selectedRegionIndex === -1) {
+    if (selectedRegionIndex === -1 && delegateRegions.length > 0) {
```

Comment:
I realized that we need to check if `delegateRegions` is not empty; otherwise, `activeRegion` will always be the first element of the `delegateRegions` array.